src/pages/ReferralV2/CopyHelper.tsx

```diff
@@ -0,0 +1,19 @@
+import useCopyClipboard from 'hooks/useCopyClipboard'
+import React from 'react'
+import { CheckCircle, Copy } from 'react-feather'
+import useTheme from 'hooks/useTheme'
+
+export default function CopyHelper({ textToCopy }: { textToCopy: string }) {
```

Comment:
we already have `CopyHelper`
Please re-use component, If design use different copy icon, just raise with designer that we need a consistent copy icon